Source/Control/adsr.h

```diff
@@ -31,31 +31,45 @@ enum
 Original author(s) : Paul Batchelor
```

Comment:
Can we change the enum to an enum class? The current version looks very C-ish.
Maybe an embedded enum like this:

```cpp
class Adsr 
{
public:
    enum class Stage
    {
        idle,
        attack,
        decay,
        sustain,
        release
    };
};
```

Reply:
I like the idea.

Reply:
To avoid API changes, which may break existing projects, I decided against it.